lib/screens/contributors.dart

```diff
@@ -23,24 +23,30 @@ class _ContributorsState extends State<Contributors>
   @override
   void initState() {
     super.initState();
-    getContributors(username: 'IEEE-VIT', repository: 'hacktoberfest-flutter')
-        .then((cards) {
-      setState(() {
-        cardList = cards;
-      });
+    fetchInitialContributors(); // Fetch and update cardList
+  }
+
+  Future<void> fetchInitialContributors() async {
+    List<ContributorCard> cards = await getContributors(
+        username: 'IEEE-VIT', repository: 'hacktoberfest-flutter');
+    setState(() {
+      cardList = cards;
+    });
+  }
+
+  void addContributor(ContributorCard newContributor) {
+    setState(() {
+      cardList.add(newContributor);
     });
   }
 
   @override
-  // ignore: must_call_super
   Widget build(BuildContext context) {

```

Comment:
This method overrides a method annotated as '@mustCallSuper' in 'AutomaticKeepAliveClientMixin', but doesn't invoke the overridden method.

fix: super.build(context); after line 44